deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+      portocol: TCP
+      internal: 6200
+    - name: apipubport
+      portocol: TCP
+      internal: 6106
+    bpm:
+      processes:
+        - name: apiserver
+          executable: /bin/bash
+          args:
+          - -c
+          - |
+            /var/vcap/jobs/apiserver/bin/apiserver start
+    run:
+      healthcheck:
+        apiserver:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6200
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=servicebroker/properties/quarks?
+  value:
+    bpm:
+      processes:
+        - name: servicebroker
+          executable: /bin/bash
+          args:
+          - -c
+          - |
+            /var/vcap/jobs/servicebroker/bin/servicebroker start      
+- type: replace
+  path: /instance_groups/name=postgres_autoscaler/jobs/name=postgres/properties/quarks?
+  value:
+    ports:
+    - name: "postgres"
+      protocol: "TCP"
+      internal: 5432
+    pre_render_scripts:
+      jobs:
+      - |
+        sed -i "s/sysctl/#sysctl/g" /var/vcap/all-releases/jobs-src/postgres/postgres/templates/pre-start.sh.erb
+    bpm:
+      processes:
+        - name: postgres
+          persistent_disk: true
+          executable: /bin/sh
+          args:
+          - -c
+          - | 
+            # set -x
+            source /var/vcap/jobs/postgres/bin/pgconfig.sh
+            source /var/vcap/jobs/postgres/bin/utils.sh
+            wait_for_file() {
+              local file_path="$1"
+              local timeout="${2:-30}"
+              until [[ -f "${file_path}" ]] || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$(expr $timeout - 1); done
+              if [[ "${timeout}" == 0 ]]; then return 1; fi
+                return 0
+            }
+            cp /var/vcap/jobs/postgres/config/postgresql.conf /var/vcap/jobs/postgres/config/pg_hba.conf /var/vcap/jobs/postgres/config/pg_ident.conf /var/vcap/store/postgres/postgres-9.6.6
+            su - vcap -c "/var/vcap/packages/postgres-9.6.6/bin/pg_ctl -D /var/vcap/store/postgres/postgres-9.6.6 -l /var/vcap/sys/log/postgres/postgres.log start"
+            sleep 300
+            /var/vcap/packages/postgres-9.6.6/bin/psql -U vcap -d postgres -f /var/vcap/jobs/postgres/config/roles.sql
+            /var/vcap/packages/postgres-9.6.6/bin/psql -U vcap -d postgres -c 'create database autoscaler;'
+            pid_file="/var/vcap/sys/run/postgres/postgres.pid"
+            log_file="/var/vcap/sys/log/postgres/postgres.log"
+            wait_for_file "${pid_file}" || {
+              echo "${pid_file} did not get created"
+              exit 1
+            }
+            wait_for_file "${log_file}" || {
+              echo "${log_file} did not get created"
+              exit 1
+            }
+            tail \
+              --pid $(cat "${pid_file}") \
+              --follow "${log_file}"
+
+#replace .ca to .certificate
```

Comment:
Since the original values are being added in this very same PR and we have control over the original values, it doesn't make sense to have those replaces here. Just set them above.

Reply:
Removed these replaces and modified the above original values.